deploy-manage/security/secure-settings.md

```diff
@@ -16,232 +15,177 @@ mapped_pages:
+:::{important}
+Secure settings must be specified on every node, and must have the same values across the cluster to ensure consistent behavior.
 
-:::{dropdown} Using the API
+Changes to the keystore take effect only after restarting {{es}}, except for [reloadable settings](#reloadable-secure-settings) that can be refreshed using the API.
+:::
 
-**Steps**
+**Create the keystore**
 
-Create the keystore:
+To create the `elasticsearch.keystore`, use the `create` command:
 
 ```sh
-curl -k -X PATCH -H "Authorization: ApiKey $ECE_API_KEY" https://$COORDINATOR_HOST:12443/api/v1/deployments/$DEPLOYMENT_ID/elasticsearch/$REF_ID/keystore \
-{
-  "secrets": {
-    "s3.client.CLIENT_NAME.access_key": {
-      "as_file": false
-      "value": "ACCESS_KEY_VALUE"
-    }
-    "s3.client.CLIENT_NAME.secret_key": {
-      "value": "SECRET_KEY_VALUE"
-    }
-  }
-}
+bin/elasticsearch-keystore create -p
 ```
 
-`ELASTICSEARCH_CLUSTER_ID`
-:   The {{es}} cluster ID as shown in the Cloud UI or obtained through the API
+You are prompted to enter the keystore password. A password-protected `elasticsearch.keystore` file is created alongside the `elasticsearch.yml` file.
 
-List the keys defined in the keystore:
+**List settings in the keystore**
 
-```sh
-{
-  "secrets": {
-    "s3.client.CLIENT_NAME.access_key": {
-      "as_file": false
-    },
-    "s3.client.CLIENT_NAME.secret_key": {
-      "as_file": false
-    }
-  }
-}
-```
-
-Create the credentials for an S3 or Minio repository:
+To list the settings in the keystore, use the `list` command.
 
 ```sh
-curl -k -X PUT -H "Authorization: ApiKey $ECE_API_KEY" https://$COODINATOR_HOST:12443/api/v1/clusters/elasticsearch/$ELASTICSEARCH_CLUSTER_ID/_snapshot/s3-repo
-{
-  "type": "s3",
-  "settings": {
-    "bucket": "s3_REPOSITORY_NAME",
-    "client": "CLIENT_NAME",
-    "base_path": "PATH_NAME"
-  }
-}
+bin/elasticsearch-keystore list
 ```
+If the {{es}} keystore is password protected, you are prompted to enter the password.
+
+**Add secure settings to the keystore**
 
-Create the credentials for a GCS repository:
+Sensitive string settings, like authentication credentials for Cloud plugins, can be added with the `add` command:
 
 ```sh
-curl -k -X PUT -H "Authorization: ApiKey $ECE_API_KEY" https://$COORDINATOR_HOST:12443/api/v1/clusters/elasticsearch/ELASTICSEARCH_CLUSTER_ID/_snapshot/s3-repo
-{
-  "type": "gcs",
-  "settings": {
-    "bucket": "BUCKET_NAME",
-    "base_path": "BASE_PATH_NAME",
-    "client": "CLIENT_NAME"
-  }
-}
+bin/elasticsearch-keystore add the.setting.name.to.set
 ```
 
-To use GCS snapshots, the cluster must have the `repository-gcs` plugin enabled.
+You are prompted to enter the value of the setting. If the {{es}} keystore is password protected, you are also prompted to enter the password.
```

Comment:
should specify the restart requirement here possibly

Reply:
It's in an `important` note just at the beginning of the tab. I think it's not needed, and if we add it in we should add it in multiple places (adding and removing settings).